data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			`	They pass along a message from the consul, who thanks you for all your efforts, and asks that you seek out his agents on <planet>, should you acquire any significant information on the Quarg ships or ringworlds. The Heliarchs say their goodbyes, and head on their way.`
+
+mission "Keep Zug Wardragon"
+	landing
+	invisible
+	source "Ring of Wisdom"
+	to offer
+		or
+			has "Heliarch Recon 3-B: done"
+			has "Heliarch Recon 3-C: done"
+	npc kill
+		government "Quarg"
+		personality heroic staying uninterested
+		system "Zubeneschamali"
+		ship "Quarg Wardragon" "Gloref Esa Kurayi"
+
+
+
+mission "Heliarch Expedition 1"
+	name "Extraterritorial Reconnaissance"
+	description "Head through the wormhole in Deneb, land on the planet there, and enter the system with the broken ringworld to get data for the Heliarch."
+	minor
+	landing
+	source "Ring of Wisdom"
+	stopover "Ruin"
+	waypoint "World's End"
+	to offer
+		has "Ruin: Landing: offered"
+		or
+			has "Heliarch Recon 3-B: done"
+			has "Heliarch Recon 3-C: done"
+	on offer
+		conversation
+			`Landing on the <origin> you remember that the Heliarchs have asked you to share with them any information you get on Quarg ships and ringworlds. Would you like to tell them about the destroyed ring you saw on the other side of the wormhole in Deneb?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+				`	(No, they don't need to know.)`
+					decline
+			`	You flag down a group of Heliarch agents who were walking by, and they salute you. "A valuable friend, you've proven to be, Captain," a Saryd of the group says. "Of assistance, how can we be?"`
+			choice
+				`	"Can you bring me to a higher-up? I have some information on ringworlds."`
+				`	"Is there anyone I can talk to about a broken ringworld?"`
+			`	The group discusses among themselves for a bit, then they bring you to a restricted section of the ringworld, reserved for Heliarchs. "Wait here, please," they tell you, offering you a seat on one end of a long table.`
+			`	Some ten minutes pass before anyone comes in, but after that many Heliarch members start filling up the room, taking their own seats at the table. When the door opens for the last of them to come in, the guard announces "Consul Aulori," to which all the Helliarchs stand up and salute, the noise of conversation giving room to hooves hitting the ground. He is an old Saryd of gray eyes, pale hairs, with a short, well-kept beard, wearing a peculiar necklace with four large gemstones hanging from it. He sits opposite to you, the other Heliarchs sitting back down when he does.`
+			`	"Forgive me for the delay you must, colleagues," he says to the other Heliarchs. "Caught up in my evening prayers, I was." He examines the room for a while, briefly glancing at each of the Heliarchs, until his eyes settle on yours. "Told, I was, that some valuable information for us, you have, Captain <last>."`
```

Comment:
Who in the everloving is a *Heliarch* praying to?

Reply:
There's the Pilgrimage mission chain, also with a Saryd, that goes some bits into Saryd religiousness and practices, if you want to check it out.

The Heliarchy is a meritocracy, yes. Whoever you are or what you believe doesn't matter to them, as long as you're loyal to the Heliarchy, and earn your rank.